src/bust.py

```diff
@@ -165,7 +165,8 @@ async def finish(self, say_goodbye: bool = True) -> None:
             await bot_member.edit(nick=self.original_bot_nickname)
 
         if say_goodbye:
-            embed_title = "❤️‍🔥 That's it everyone ❤️‍🔥"
+            goodbye_emoji = ":heart_on_fire:"
+            embed_title = f"{goodbye_emoji} That's it everyone {goodbye_emoji}"
```

Comment:
I'm not sure if separating `:heart_on_fire:` into a separate variable buys us much here?

Reply:
It doesn't save us any characters, my reasoning was that it visually separates it from the text in syntax highlighting, and that if we ever theoretically wanted to change it it would be in one location. It's probably pretty arbitrary which is better though